server/schedule/placement/config.go

```diff
@@ -0,0 +1,160 @@
+// Copyright 2020 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package placement
+
+import (
+	"bytes"
+	"encoding/json"
+)
+
+// ruleConfig contains rule and rule group configurations.
+type ruleConfig struct {
+	rules  map[[2]string]*Rule
+	groups map[string]*RuleGroup
```

Comment:
Please add the comment to explaining the meaning of the key, like:
// rules {groupID,ruleID} -> Rule

Reply:
Good idea.